sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -138,6 +138,17 @@ extern "C" {
 #endif
 #endif
 
+/**
+ * @brief   Include a Route Information Option for subnets
+ *          on other interfaces in the last Router Advertisement
+ *          generated by @ref gnrc_ipv6_nib_change_rtr_adv_iface
+ *
+ *          Requires the `gnrc_ipv6_nib_rio` module.
+ */
+#ifndef CONFIG_GNRC_IPV6_NIB_RIO_IN_LAST_RA
+#define CONFIG_GNRC_IPV6_NIB_RIO_IN_LAST_RA          0
```

Comment:
```suggestion
#define CONFIG_GNRC_IPV6_NIB_ONLY_RIO_IN_LAST_RA     0
```

?

Reply:
Doesn't that imply that RIOs are sent by mechanisms *other* than last / empty RA?
Currently this would be the only place where we are *sending* RIOs.

Reply:
I wanted to imply, that with this option activated _only_ a RIO is added (and no other options) to the final RA. If you can think of a better name to express this unambiguously, please go ahead and use that.

> Currently this would be the only place where we are _sending_ RIOs.

The word "currently" is key here ;-) That might change.

Reply:
Ah ok now I see what you mean. It wasn't my first association since the last RA currently doesn't contain any option, so to me putting an *only* there sounds like it would exclude other options, when in fact it *adds* an option.

How about `CONFIG_GNRC_IPV6_NIB_ADD_RIO_IN_LAST_RA`?

Reply:
> How about `CONFIG_GNRC_IPV6_NIB_ADD_RIO_IN_LAST_RA`?

Now that I remember my own code again :+1: